BlazarService/src/main/java/com/hubspot/blazar/util/SingularityBuildLauncher.java

```diff
@@ -31,18 +29,11 @@ public synchronized void launchBuild(ModuleBuild build) throws Exception {
   private SingularityRunNowRequest buildRequest(ModuleBuild build) {
     String buildId = Long.toString(build.getId().get());
 
-    Optional<com.hubspot.mesos.Resources> mesosResources = Optional.absent();
-    Optional<Resources> blazarResources = build.getBuildConfig().get().getBuildResources();
```

Comment:
This is the line that broke things.
Queued ModuleBuilds don't have a build config _yet_.